isis/src/base/apps/isisimport/isisimport.cpp

```diff
@@ -182,8 +183,15 @@ namespace Isis {
     importer.SetBase(base);
     importer.SetMultiplier(multiplier);
 
-    // TODO: how to handle this?
-    importer.SetFileHeaderBytes(0);
+    // Handle PDS3 header
+    if (jsonData.contains("^IMAGE")) {
+      std::string offset = jsonData["^IMAGE"]["Value"];
+      std::string recBytes = jsonData["RECORD_BYTES"]["Value"];
+      importer.SetFileHeaderBytes((std::stoi(offset) - 1) * std::stoi(recBytes));
+    }
+    else {
+      importer.SetFileHeaderBytes(0);
+    }
```

Comment:
I think there may be a better way to handle this that I would like to talk to you and Stuart about

Reply:
I originally tried grabbing these within the template but I kept running into a vague json error. The only thing I could determine was that the template does not like {{ ^IMAGE.Value }} due to the caret. In the end, I think you would still have to check if the values are in the template, resulting in something similar to this. But I am sure there another way that I have not thought of yet.